scripts/variantstore/wdl/GvsCallsetCost.wdl

```diff
@@ -110,10 +110,10 @@ task CoreStorageModelSizes {
             local table_pattern="$1"
             local output_file_name="$2"
 
-            bq --apilog=false query --project_id='~{project_id}' --format=csv --use_legacy_sql=false \
-                "SELECT round(sum(total_billable_bytes) / (1024*1024*1024),2) \
-                    FROM \`~{project_id}.~{dataset_name}.INFORMATION_SCHEMA.PARTITIONS\` \
-                    WHERE table_name LIKE '${table_pattern}'" | tail -1 > ${output_file_name}
+            bq --apilog=false query --project_id=~{project_id} --format=csv --use_legacy_sql=false \
+                'SELECT round(sum(total_billable_bytes) / (1024*1024*1024),2) \
```

Comment:
i almost want this as a constant instead of all the math, but I'm not gonna die on that hill